torch/testing/_internal/common_quantization.py

```diff
@@ -1633,6 +1635,23 @@ def forward(self, x):
         return self.dequant(x)
 
 class ManualLinearQATModel(torch.nn.Module):
+    r"""A Module with manually inserted `QuantStub` and `DeQuantStub`
+    """
+    def __init__(self, qengine):
+        super().__init__()
+        self.qconfig = torch.quantization.get_default_qat_qconfig(qengine)
+        self.quant = QuantStub()
+        self.dequant = DeQuantStub()
+        self.fc1 = torch.nn.Linear(5, 1).to(dtype=torch.float)
+        self.fc2 = torch.nn.Linear(1, 10).to(dtype=torch.float)
+
+    def forward(self, x):
+        x = self.quant(x)
+        x = self.fc1(x)
+        x = self.fc2(x)
+        return self.dequant(x)
+
+class ManualDropoutQATModel(torch.nn.Module):
```

Comment:
we can remove the two linears in the this model I think

Reply:
remove linear op create an empty parameter list, optimizer would fail.

Reply:
optional nit: I see, can we move one of the linears then?

Reply:
fc2 removed